src/components/api-response.js

```diff
@@ -308,6 +311,13 @@ export default class ApiResponse extends LitElement {
     `;
   }
 
+  exampleValueTemplate(example) {
```

Comment:
I'm not a fan of having this inline here, we have an explicit check for `json` on the outside above the this template insertion, it's weird to then have this template explicitly check for the type again.

Also, here's something I'm not sure of, above we are checking for `exampleFormat === json`, I'm not entirely sure why it would inconsistent here. It's been a while since I looked at this, but some sort of consistency would be in how we check these.

Reply:
I agree.

The code is duplicated between the branch "only one example" and "multiple examples". But it's not exactly the same because of css classes. To minimize the change, I have only handled xml in this method, but I can try to mutualise all use cases.

Reply:
I do still expect this to be a small PR and a minimized change, but I think it makes sense to move the === `json` in here, and use the same variables to decide how to format it. It's fine if it is a bunch of `if...returns`, this is probably the right time to make a subtle change to how we are rendering this, I wouldn't expect it to be large.